sample_annotator/clients/submission_portal_client.py

```diff
@@ -0,0 +1,185 @@
+import pandas as pd
+import nmdc_schema.nmdc as nmdc
+import linkml
+import numpy as np
+import linkml_runtime.dumpers.json_dumper as dumper
+import requests
+import json
+from clients.nmdc.runtime_api_client import RuntimeApiSiteClient
+from dotenv import dotenv_values
+
+
+
+#I had initially misunderstood, and did not design the code as part of an class/object so these are mostly
+#independent functions that I've added to this overarching class
+
+#API things are still missing as I only just got access to them ~1 hour before pushing this code up
+class submission_portal_client:
```

Comment:
Class names should typically be PascalCase. So you would have `SubmissionPortalClient`.

Reply:
Ive made this update in the code.